src/State.ts

```diff
@@ -20,15 +21,15 @@ export class State {
         data?: unknown;
         created?: number;
```

Comment:
How about we make `created` a required property instead of requiring a `ClockService` instance? Wouldn't storage strings always have the `created` property anyway?

Reply:
great idea, i will try that out...